armory/datasets/config_load.py

```diff
@@ -0,0 +1,85 @@
+"""
+Temporary file for testing loading from config without modifying armory.utils
+"""
+
+import copy
+
+from armory.datasets import load, preprocessing, generator
+
+
+def actual_loader(
```

Comment:
could the `ArmoryDataGenerator` object instead be instantiated inside the `load_dataset()` function, obviating the need for this extra function? I think this makes things a bit simpler if so

Reply:
I wanted to separate them, because I think this function is much closer to what scenario configs should look like, not the current function. (But yes, having both is unneeded, ultimately.)